transcom/mymove
# B-17918: remove old PPM INTERNAL APIs - SET B

## [B-17918](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A868169)

[E-04480](https://www13.v1host.com/USTRANSCOM38/Epic.mvc/Summary?oidToken=Epic%3A833435)


## Summary
This PR is for the deprecation of the following INTERNAL APIs:
`/personally_procured_move/{personallyProcuredMoveId}/expense_summary - GET`

- http://milmovelocal:3000/swagger-ui/internal.html#/ppm/requestPPMExpenseSummary

`/personally_procured_moves/{personallyProcuredMoveId}/create_ppm_attachments - POST`

- http://milmovelocal:3000/swagger-ui/internal.html#/ppm/createPPMAttachments

NOTE: All references to the deprecated APIs were removed both on the GO and client-side. To mitigate risk, all targeted APIs marked for deprecation were grouped into two branches for [B-17918](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A868169): 

- B-17918-Remove-create_ppm_attachments-from-Internal-API-A ([PR](https://github.com/transcom/mymove/pull/11675))

- B-17918-Remove-create_ppm_attachments-from-Internal-API-B

**This PR is for branch: B-17918-Remove-create_ppm_attachments-from-Internal-API-B**

The [deprecation process](https://transcom.github.io/mymove-docs/docs/api/guides/how-to-deprecate-endpoints/) was not observed because it's INTERNAL and not Prime ([per Dre](https://caci-bits.slack.com/archives/C05DX3AG6EL/p1701194838343559)).

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test
The following have been removed from Swagger:

`/personally_procured_move/{personallyProcuredMoveId}/expense_summary - GET`

- http://milmovelocal:3000/swagger-ui/internal.html#/ppm/requestPPMExpenseSummary

`/personally_procured_moves/{personallyProcuredMoveId}/create_ppm_attachments - POST`

- http://milmovelocal:3000/swagger-ui/internal.html#/ppm/createPPMAttachments
1. Access Internal Swagger: http://milmovelocal:3000/swagger-ui/internal.htm
2. http://milmovelocal:3000/swagger-ui/internal.html#/ppm/requestPPMExpenseSummary (does not exist)
3. http://milmovelocal:3000/swagger-ui/internal.html#/ppm/createPPMAttachments  (does not exist)
4. Log into http://milmovelocal:3000/  (not see any React build errors, or runtime errors navigating the application)
### Frontend


### Backend


## Screenshots
The following have been removed from Swagger:

`/personally_procured_move/{personallyProcuredMoveId}/expense_summary - GET`

- http://milmovelocal:3000/swagger-ui/internal.html#/ppm/requestPPMExpenseSummary

`/personally_procured_moves/{personallyProcuredMoveId}/create_ppm_attachments - POST`

- http://milmovelocal:3000/swagger-ui/internal.html#/ppm/createPPMAttachments
